styx-common/src/main/java/com/spotify/styx/state/TriggerSerializer.java

```diff
@@ -59,8 +60,8 @@ public Trigger deserialize(ByteString json) {
   private static class TriggerSerializerVisitor implements TriggerVisitor<PersistentTrigger> {
 
     @Override
-    public PersistentTrigger natural(String triggerId) {
-      return new PersistentTrigger("natural", triggerId);
+    public PersistentTrigger natural() {
+      return new PersistentTrigger("natural", NATURAL_TRIGGER_ID);
```

Comment:
The trigger id is ignored in this case right?

The json produced for the natural trigger should just be `{'@type':'natural'}`.